README.md

```diff
@@ -60,3 +60,23 @@ docker exec -i <keycloak-container-name> bash /usr/scripts/bootstrap-keycloak.sh
 ```
 
 The keycloak admin portal can be accessed in the browser by navigating to localhost and the port specified by the `KEYCLOAK_PORT` env var, e.g., `localhost:9821`
+
+## Mongo
+
+Annotations can be imported into mongo using the following command. Note that that the headers should not be included in the csv and the order of the fields passed to the `fields` argument should match the order of the fields in the csv.
+
+```bash
+mongoimport --db=annotations --collection=annotations --type=csv \
+   --columnsHaveTypes \
+   --fields="pos.int32(),ref.string(),alt.string(),chrom.string(),nhomalt.int32(),an.int32(),af.double(),assembly.string()" \
+   --file=<filename>.csv \
+   -u <username> --password=<pass> --authenticationDatabase=admin
+```
+
+Then make sure to create the following indexes:
+
+```
+db.annotations.createIndex({"pos": 1})
+db.annotations.createIndex({"assembly":1})
+db.annotations.createIndex({"alt": 1, "chrom": 1, "pos": 1, "ref": 1 })
```

Comment:
This can also be ran with a single command as `db.annotations.createIndexes([
{"pos": 1}, {"assembly":1}, {"alt": 1, "chrom": 1, "pos": 1, "ref": 1 }
])`